django/db/backends/oracle/creation.py

```diff
@@ -77,9 +77,6 @@ def _create_test_db(self, verbosity=1, autoclobber=False, keepdb=False):
             try:
                 self._create_test_user(cursor, parameters, verbosity, keepdb)
             except Exception as e:
-                # If we want to keep the db, then we want to also keep the user.
-                if keepdb:
-                    return
```

Comment:
Ok, I prepared separate [PR](https://github.com/django/django/pull/7531) for this.